Don't print error message when there's no system-provided crystal.

--- a/bin/crystal
+++ b/bin/crystal
@@ -150,7 +150,7 @@
 if [ -z "$CRYSTAL_CONFIG_LIBRARY_PATH" ] || [ -z "$CRYSTAL_LIBRARY_PATH" ]; then
   CRYSTAL_INSTALLED_LIBRARY_PATH="$(
       export PATH="$(remove_path_item "$(remove_path_item "$PATH" "$SCRIPT_ROOT")" "bin")"
-      crystal env CRYSTAL_LIBRARY_PATH || echo ""
+      crystal env CRYSTAL_LIBRARY_PATH 2>/dev/null || echo ""
     )"
   export CRYSTAL_LIBRARY_PATH=${CRYSTAL_LIBRARY_PATH:-$CRYSTAL_INSTALLED_LIBRARY_PATH}
   export CRYSTAL_CONFIG_LIBRARY_PATH=${CRYSTAL_CONFIG_LIBRARY_PATH:-$CRYSTAL_INSTALLED_LIBRARY_PATH}
